src/main/java/Duke.java

```diff
@@ -1,10 +1,91 @@
+import java.util.Arrays;
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
+    public static void printHorizontalLine() {
+        String horizontal_line = "____________________________________________________________";
+        System.out.println(horizontal_line);
+    }
+
+    public static void welcome() {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        printHorizontalLine();
+        System.out.println(" Hello! I'm Duke");
+        System.out.println(" What can I do for you?");
+        printHorizontalLine();
+    }
+
+    public static void markAsDone(Task[] tasks, String line) {
+        String[] input = line.split(" ");
+        int index = Integer.parseInt(input[1]) - 1;
+        tasks[index].markAsDone();
+        printHorizontalLine();
+        System.out.println(" Nice! I've marked this task as done:");
+        System.out.println(" " + tasks[index].toString());
+        printHorizontalLine();
+    }
+
+    public static void printList(Task[] tasks, int currCount) {
+        printHorizontalLine();
+        System.out.println(" Here are the tasks in your list:");
+        Task[] taskList = Arrays.copyOf(tasks, currCount);
+        int count = 1;
```

Comment:
Maybe we can add some notes to note which count it is. e.g. task_count

Reply:
Thanks!